tests/integration/support/saveEdit.js

```diff
@@ -0,0 +1,6 @@
+Cypress.Commands.add('saveEdit', () => {
+  cy.get('.edit-form')
+    .find('.header-actions')
+    .contains('ui5-button:visible', 'Save')
+    .click();
+});
```

Comment:
it's almost the same as create command, please make it as one command, with prop if it's save or create